CDCgov/NEDSS-Modernization
# Demographics/refresh

## Description
Setting notifyOnNetworkStatusChange to make refetch work in order to see displayed the actual changes and latest updates in the patient profile summary card .

## Tickets
https://cdc-nbs.atlassian.net/browse/CNFT1-1639

## Steps to verify


1. Navigate to any patient profile
2. Navigate to the demographics tab
3.  change any information either in the tables or in the cards (example: change an address) 
4. see the updated information in the summary card at the top of the patient profile
